### PR TITLE
Implementing tslint checks as an npm command and pre-commit hooks as well

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
eslint does not scan `.tsx` files. Hence, we need `tslint`. Same is being done in this PR.

### CODE REVIEW

**Comment by ganeshpatro321** [comment]
@Harkishen-Singh Please check https://github.com/typescript-eslint/tslint-to-eslint-config/ and see whether this helps us. I tried out this, but its showing some error, couldnt figure it out

File: src/components/layouts/Header.tsx [hunk 1]
```diff
@@ -16,12 +16,12 @@ export default function Header() {
...
       <header>
-        <div className="logo-name">Bench-routes</div>
+        <div className='logo-name'>Bench-routes</div>
         {!showNotification ? (
-          <div className="notification-icon">
+          <div className='notification-icon'>
             <img
-              src="assets/icons/notify-icon.svg"
-              alt="notification"
+              src='assets/icons/notify-icon.svg'
```
Comment (ganeshpatro321) [inline src/components/layouts/Header.tsx]: Eslint made single-quotes to double quotes, tslint making it to single quotes XD

Comment (Harkishen-Singh) [reply src/components/layouts/Header.tsx]: yeah..we need to unify some rules.

**Comment by ganeshpatro321** [comment]
That file didint help? @Harkishen-Singh Are we sticking to tslint

**Pull request merged**